index.html

```diff
@@ -0,0 +1,304 @@
+          <a
+            href="#"
+            class="header__logo">
+            <img
+              class="header__logo-img"
+              src="./images/logo.svg"
+              alt="nova_logo" />
+          </a>
+          <nav class="header__nav nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./#team"
+                  >The Team</a
+                >
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./#pricing"
+                  >Pricing</a
+                >
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./#features"
+                  >Features</a
+                >
+              </li>
+              <li class="nav__item">
+                <button class="nav__button btn btn-blue">Download FREE!</button>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <section class="hero row">
+        <div class="hero__container container">
+          <article class="hero__content">
+            <h1 class="hero__header">Beautiful Free Nova template</h1>
+            <p class="hero__text">
+              A top notch premium quality template for your next web project.
+            </p>
+            <button class="hero__btn btn btn-green">Download FREE!</button>
+          </article>
+          <article class="hero__image">
+            <div class="hero__image-container">
+              <img
+                class="hero__img"
+                src="./images/screen.png"
+                alt="" />
+            </div>
+          </article>
+        </div>
+      </section>
+    </div>
+    <section
+      id="features"
+      class="features row">
+      <h2 class="features__header container">FEATURES</h2>
+      <div class="features__items container">
+        <div class="features__item">
+          <div class="features__item-img">
+            <img
+              src="./images/responsive.svg"
+              alt="mobile" />
+          </div>
+          <h3 class="features__item-title">FULLY RESPONSIVE</h3>
+          <p class="features__item-text">
+            Looks amazing on any device: smartphone, tablet, laptop and desktop.
+          </p>
+        </div>
+        <div class="features__item">
+          <div class="features__item-img">
+            <img
+              src="./images/customizable.svg"
+              alt="tools" />
+          </div>
+          <h3 class="features__item-title">CUSTOMIZABLE</h3>
+          <p class="features__item-text">
+            Change the colors, pictures or any of the sections to suit your
+            needs.
+          </p>
+        </div>
+        <div class="features__item">
+          <div class="features__item-img">
+            <img
```

Comment:
Może zamiast wszędzie powielać `item-` to zrobić z tego osobny block? np.:
```
     <div class="features__item item"> <!-- nowy Block -->
          <div class="item__img"> <!-- Elemenet dla nowego Blocku -->
```

Reply:
Nie robiłam oddzielnego bloku, ponieważ poniżej w kodzie mam znowu sekcje pricing__item i żeby rozróżnić te klasy zostawiłam .pricing__item-title i .features__item-title